Initial docs for globals

rubennorte opened the pull request:
This adds placeholder docs for globals, in a new section under the `APIs` page:


<img width="1248" height="999" alt="Screenshot 2025-09-09 at 18 51 47" src="https://github.com/user-attachments/assets/987b2c5c-3933-413b-b080-b7f3d6c56032" />

Simek commented:
We currently use "🚧" emoji to indicate deprecated components/APIs. 

<img width="1708" height="532" alt="Screenshot 2025-09-09 at 19 03 56" src="https://github.com/user-attachments/assets/85370759-c1fa-4663-9371-50f6956f6a73" />

Recently, we started to annotate experimental pages with "⚗️" emoji, maybe this would fit better for the WIP pages?

<img width="1812" height="586" alt="Screenshot 2025-09-09 at 19 05 14" src="https://github.com/user-attachments/assets/c41a5308-ec22-4e8e-8ea6-d8d9764ce565" />

Simek commented on website/sidebars.ts:
```diff
@@ -221,6 +221,44 @@ export default {
+      {
+        type: 'category',
+        label: 'Globals',
+        collapsed: false,
```
```suggestion
        collapsed: true,
```

Due to shear amount of the pages, we might want to collapse the section by default, to avoid large scrollable sidebar initially. We might want to collapse also further groups, to match what we do in "Guides".

Alternatively, if we want to retain visibling, we could move the "Globals" section at the end of sidebar.

rubennorte commented:
> We currently use "🚧" emoji to indicate deprecated components/APIs.

That's an unfortunate use of that emoji. Maybe we should reconsider that and use 🗑️ for deprecated ones, as MDN does:

<img width="180" height="108" alt="Screenshot 2025-09-09 at 18 28 19" src="https://github.com/user-attachments/assets/1d8762df-53cc-4bbc-8b69-4c710d05e63b" />

rubennorte commented:
@Simek we changed the existing pages for deprecated APIs to use the 🗑️ emoji and moved the 🚧 emoji to the banner instead of in the title in these cases.

Simek commented:
@rubennorte Yup, that's a good idea, I like it! 👍 

We also use "❌" emoji for the deleted/removed parts from the core, which still had a pages avaible on the website, and with "🗑️" emoji we can still retain this distinction.
